src/python/T0/RunConfig/RunConfigAPI.py

```diff
@@ -53,6 +53,39 @@ def extractConfigParameter(configParameter, era, run):
     else:
         return configParameter
 
+def filterStreams(isMainAgent = True, secondaryAgentStreams = [], hltStreamMapping=None, streamers=None):
```

Comment:
I'm not sure why this function is part of RunConfigAPI. It's not setting any configuration nor is it used by any of the other functions on the API.

Reply:
This was the big question I ran into yesterday. I decided to put it there because it allows Tier0FeederPoller and StorageManagerAPI to access it. I ran into importing problems when I tried defining it in the Tier0FeederPoller. Alternatives I thought are:

* Define it in the Tier0FeederPoller
   * Split [StorageManagerAPI.injectNewData](https://github.com/dmwm/T0/blob/master/src/python/T0/StorageManager/StorageManagerAPI.py#L15) so the Tier0FeederPoller can have access to the [newData](https://github.com/dmwm/T0/blob/master/src/python/T0/StorageManager/StorageManagerAPI.py#L68) variable and filter it. I believe this sounds like the best option, but requires heavier modifications, so I wanted your opinion on it.
* Define it in the StorageManagerAPI
   * This works, and is neat, and Tier0FeederPoller would have access to it to filter out hltConfig data. The inconsistency is that it would also be used for hlt data. Perhaps that is not a problem at all

@germanfgv